<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>pageSetupDialog_q</class>
  <widget class="QDialog" name="pageSetupDialog_q" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>248</width>
        <height>199</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Page Setup</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="QCheckBox" name="newPageForSection" >
          <property name="text" >
            <string>start each section on a new page</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QCheckBox" name="printHeader" >
          <property name="text" >
            <string>print header on every page</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="QCheckBox" name="printLegend" >
          <property name="text" >
            <string>print legend</string>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QCheckBox" name="printObjects" >
          <property name="text" >
            <string>print objects used in rules</string>
          </property>
        </widget>
      </item>
      <item row="5" column="0" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer9" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string>Alt+O</string>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut" >
                <string>Alt+C</string>
              </property>
              <property name="autoDefault" >
                <bool>false</bool>
              </property>
              <property name="default" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="4" column="0" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="textLabel1" >
              <property name="text" >
                <string>Scale tables: </string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="tableResolution" >
              <item>
                <property name="text" >
                  <string>50%</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>75%</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>100%</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>150%</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>200%</string>
                </property>
              </item>
            </widget>
          </item>
          <item>
            <spacer name="spacer10" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction></pixmapfunction>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>pageSetupDialog_q</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>pageSetupDialog_q</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
